components/tag-list/tag-list.js

```diff
@@ -326,6 +334,12 @@ class TagList extends LocalizeCoreElement(InteractiveMixin(ArrowKeysMixin(LitEle
 	_handleClearAll(e) {
 		if (!this._items) return;
 
+		/** Dispatched when a user selects to delete all tag list items. The consumer must handle the actual element deletion and focus behaviour. */
+		this.dispatchEvent(new CustomEvent(
```

Comment:
Now that we're sending this event, should we only send https://github.com/BrightspaceUI/core/blob/main/components/tag-list/tag-list-item-mixin.js#L172-L175 if it's not a `clearAll` scenario?  To avoid consumers handling things twice if they listen to both types?

Reply:
We probably can - yeah. We would just want to clarify in the docs that if `clearable` is on the tag-list then the consumer should be listening for the `clear-all` event and handle clearing all the items from there

Basically document that whatever has `clearable` on it needs to listen to the corresponding event and handle item deletion (which actually seems fairly straightforward)